api/controller.py

```diff
@@ -452,6 +453,7 @@ def setup_configuration_dependent_controllers(self):
         This method will be called fresh every time the site
         configuration changes.
         """
+        self.basic_auth_controller = BasicAuthTempTokenController(self.auth)
```

Comment:
I would like to call this basic_auth_token_controller because this makes it sound like the only controller that responds to Basic Auth.